Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -705,14 +703,72 @@ extension SPNoteEditorViewController: InterlinkProcessorDelegate {
 }
 
 
+// MARK: - Tags
+//
+extension SPNoteEditorViewController {
+    @objc
+    func configureTagListViewController() {
+        tagListViewController = NoteEditorTagListViewController(note: note)
+        addChild(tagListViewController)
+        let tagView = tagListViewController.view!
+
+        tagView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(tagView)
+
+        NSLayoutConstraint.activate([
+            tagView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tagView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        tagListBottomConstraint = tagView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        tagListBottomConstraint.isActive = true
+
+        tagListViewController.didMove(toParent: self)
+
+        tagListViewController.delegate = self
+    }
+
+    private func updateTagListPosition() {
+        guard !isVoiceOverEnabled else {
+            tagListViewController.view.transform = .identity
+            return
+        }
+
+        let contentHeight = noteEditorTextView.contentSize.height - noteEditorTextView.textContainerInset.bottom
+        let maxContentY = noteEditorTextView.convert(CGPoint(x: 0, y: contentHeight), to: view).y
+
+        tagListViewController.view.transform = .identity
```

Comment:
Can we get rid of this line?

Reply:
We can not. We need to reset transform to get original frame without any modifications so that we can correctly calculate intersection with editor content on the next line :-)